i18n/en.json

```diff
@@ -932,6 +932,34 @@
     "translation": "shrug"
   },
   {
+    "id": "api.command_mute.desc",
+    "translation": "Mutes the notifications for this channel"
+  },
+  {
+    "id": "api.command_mute.name",
+    "translation": "mute"
+  },
+  {
+    "id": "api.command_mute.error",
+    "translation": "An error occurred on muting {{.Channel}} occurred."
+  },
+  {
+    "id": "api.command_mute.success_mute",
+    "translation": "You will not receive notifications for {{.Channel}} until channel mute is turned off."
+  },
+  {
+    "id": "api.command_mute.success_unmute",
+    "translation": "{{.Channel}} is no longer muted."
+  },
+  {
+    "id": "api.command_mute.success_mute_direct_msg",
+    "translation": "You will not receive notifications for this direct message channel until channel mute is turned off."
```

Comment:
Propose we update to: "You will not receive notifications for this channel until channel mute is turned off."